src/cpu-template-helper/src/utils.rs

```diff
@@ -19,6 +19,63 @@ pub fn add_suffix(path: &Path, suffix: &str) -> PathBuf {
     path.with_file_name(new_file_name)
 }
 
+#[cfg(target_arch = "aarch64")]
+pub mod aarch64 {
+    #[macro_export]
+    macro_rules! reg_modifier {
+        ($addr:expr, $value:expr) => {
+            RegisterModifier {
+                addr: $addr,
+                bitmap: RegisterValueFilter {
+                    filter: u128::MAX,
+                    value: $value,
+                },
+            }
+        };
+    }
+}
+
+#[cfg(target_arch = "x86_64")]
+pub mod x86_64 {
+    #[macro_export]
+    macro_rules! cpuid_reg_modifier {
+        ($register:expr, $value:expr) => {
+            CpuidRegisterModifier {
+                register: $register,
+                bitmap: RegisterValueFilter {
+                    filter: u32::MAX.into(),
+                    value: $value,
+                },
+            }
+        };
+    }
+
+    #[macro_export]
+    macro_rules! cpuid_leaf_modifier {
```

Comment:
Using `#[macro_export]` is the old approach to export macros. The new approach is `pub(crate) use cpuid_reg_modifier;` see https://stackoverflow.com/a/31749071/4301453

Reply:
I didn't know it! Thanks!